server/controllers/finance/reports/cashflow/index.js

```diff
@@ -361,6 +361,8 @@ function report(req, res, next) {
      * @param {*} source
      * @param {*} sourceTotalByTextKeys
      */
+
+  // eslint-disable-next-line no-shadow
```

Comment:
Since there is another definition of a function with this name and the code looks the same, is it possible to just reuse this function, instead of recreating it again.